### PR TITLE
fix map-data not passing through the interceptor and change the site …

### PR DESCRIPTION
**Pull request by ryanwilliamson01**
Resolves issue #9 . Quick change to fix the interceptor issue & change the site title.

### CODE REVIEW

File: frontend/src/app/feature/maskreport/maskreport.module.ts [hunk 1]
```diff
@@ -93,7 +96,7 @@ import { AgGridModule } from "ag-grid-angular";
     MatGridListModule,
     MatRadioModule
   ],
-  providers: [MaskReportService, MaskReportDetailsResolver],
+  providers: [MaskReportService, MaskReportDetailsResolver, MapDataService, PlacesService, MapService],
```
Comment (shankari) [inline frontend/src/app/feature/maskreport/maskreport.module.ts:99]: ah so this *was* the problem. Good to know it wasn't the observable v/s promise

File: frontend/src/environments/environment.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 export const environment = {
   production: false,
-  apiEndpoint: "http://localhost:3000/api",
+  apiEndpoint: "https://api.covid.k8s.transity.io",
```
Comment (shankari) [inline frontend/src/environments/environment.ts:7]: Please revert this change

File: frontend/src/index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Mean Stack Demo</title>
+    <title>Mask Reports</title>
```
Comment (shankari) [inline frontend/src/index.html:5]: Good catch!

**Comment by shankari** [comment]
@ryanwilliamson01 can you please fix this so I can merge?

File: frontend/src/environments/environment.ts (hunk 2 above)
Comment (ryanwilliamson01) [reply frontend/src/environments/environment.ts]: Done.

**Comment by ryanwilliamson01** [comment]
Changes have been made.
